test/gpu/native/atomics.chpl

```diff
@@ -5,17 +5,17 @@ config param excludeForRocm = (CHPL_GPU == "amd");
 
 // CPU versions of associated GPU functions. These aren't actually atomic but
 // are used for test verification:
-proc cpuAtomicAdd (ref x : ?T, val : T) : void { x += val; }
-proc cpuAtomicSub (ref x : ?T, val : T) : void { x -= val; }
-proc cpuAtomicExch(ref x : ?T, val : T) : void { x = val; }
-proc cpuAtomicMin (ref x : ?T, val : T) : void { x = min(x, val); }
-proc cpuAtomicMax (ref x : ?T, val : T) : void { x = max(x, val); }
-proc cpuAtomicInc (ref x : ?T, val : T) : void { x = if x >= val then 0:T else x + 1; }
-proc cpuAtomicDec (ref x : ?T, val : T) : void { x = if x == 0:T || x > val then val else x-1; }
-proc cpuAtomicAnd (ref x : ?T, val : T) : void { x &= val; }
-proc cpuAtomicOr  (ref x : ?T, val : T) : void { x |= val; }
-proc cpuAtomicXor (ref x : ?T, val : T) : void { x ^= val; }
-proc cpuAtomicCAS (ref x : ?T, cmp : T, val : T) : void { x = if x == cmp then val else x; }
+proc cpuAtomicAdd (ref x : ?T, val : T) : T { x += val; return x-val;}
+proc cpuAtomicSub (ref x : ?T, val : T) : T { x -= val; return x+val;}
```

Comment:
Nit pick, but for consistencies sake (and maybe for correctness too in case of underflow/overflow?) I'd probably just do the same `const retval = x` and `return retval` thing you do for most of the other ops here.